docs/api/focal-point.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: FocalPoint
+---
+
+The `FocalPoint` is the TypeScript interface that represents the focal point X and Y coordinates in percentage.
```

Comment:
The `Focal Point` is a TypScript interface that represents the X and Y coordinates of the focal point in percent.